python/adelphi/tests/unit/test_anonymize.py

```diff
@@ -0,0 +1,133 @@
+import pytest
```

Comment:
This import doesn't appear to be used: everything below uses the conventional Python unittest framework (which seems like the right decision to me) and I don't see any difference in behaviour when this import is removed.

Reply:
fixed